app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/UserType.scala

```diff
@@ -18,6 +18,7 @@ package uk.gov.hmrc.cgtpropertydisposalsfrontend.models
 
 import julienrf.json.derived
 import play.api.libs.json.OFormat
+//import cats.Eq
```

Comment:
can we remove this if it's not needed?

Reply:
Will work refactoring these two specs separately (4 tests are ignored because of agents journey is missing some logic)
ViewReturnControllerSpec
 2. CheckAllAnswersAndSubmitControllerSpec